### DISCUSSION TITLE
Help using the tool

### DISCUSSION THREAD

**Discussion by fedejsoren** [yuezk/GlobalProtect-openconnect#196] [Q&A]
Hello! 
I was trying to use this to connect to a globalProtect and I get a 
![image](https://user-images.githubusercontent.com/2771497/203797370-72508b91-86ec-4866-91f1-c9a7ae2dac2d.png)
and sadly the error is not too descriptive for me to find out how to proceed..

``` 
Unknown response for gateway prelogin interface.
```

Do you have any pointers for me? :)

Thank you!

**Comment by yuezk** [comment]
Thanks for your feedback.

Normally, I would like to request the logs. You can collect the logs by running the `gpclient` in your terminal.
Another workaround is to try this client https://github.com/dlenski/gp-saml-gui, if it works, then it should be a bug of my client.
